### PR TITLE
Don't try to read from directories.

### PR DESCRIPTION
**Pull request by Nonsense331**

### CODE REVIEW

**Comment by deploybot2000** [comment]
NO PIVOTAL CASES DETECTORED

File: lib/crantastic.rb [hunk 1]
```diff
@@ -146,12 +146,10 @@ def add_version_to_db(pkg, crantastic_package_id)
...
       data = {}
       files.each do |f|
-        if File.exists?(pkgdir + f)
+        if File.exists?(pkgdir + f) && !File.directory?(pkgdir + f)
           data[f.downcase.to_sym] = File.read(pkgdir + f)
-        else
-          if File.exists?(pkgdir + "inst/" + f)
-            data[f.downcase.to_sym] = File.read(pkgdir + "inst/" + f)
-          end
+        elsif File.exists?(pkgdir + "inst/" + f) && !File.directory?(pkgdir + "inst/" + f)
```
Comment (boatrite) [inline lib/crantastic.rb]: Interpolate `"#{pkgdir}inst/#{f}"`

**Comment by boatrite** [comment]
:boat:

File: lib/crantastic.rb (hunk 1 above)
Comment (petitJAM) [inline lib/crantastic.rb]: probably `File.join`?

**Pull request merged**